redminelib/engines/base.py

```diff
@@ -99,7 +99,9 @@ def bulk_request(self, method, url, container, **params):
                 for num in range(limit - self.chunk, 0, -self.chunk):
                     offset += self.chunk
                     limit -= self.chunk
-                    bulk_params.append({'offset': offset, 'limit': limit})
+                    bulk_param = dict(params)
+                    bulk_param.update({'offset': offset, 'limit': limit})
+                    bulk_params.append(bulk_param)
```

Comment:
Can be rewritten as ``bulk_params.append(dict(params, **{'offset': offset, 'limit': limit}))``